apps/andi/lib/andi/input_schemas/organizations.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Andi.InputSchemas.Organizations do
```

Comment:
I find this module name a little confusing as it's getting and manipulating `HarvestedDatasets` instead of `Organizations`. Maybe if we want to keep this module name, we can rename some of the functions to reflect that? I can also change that when I add the organization table to Ecto

Reply:
This is something @bbrewer-forge and I discussed. All the harvesting stuff is going to live alongside the organization code. There was no point in calling this something else related to harvesting then going back and changing it when the organizations page gets developed

Reply:
Yeah, hopefully this will get more fleshed out when we move the organization to the repo